data/bookkeeping/block.go

```diff
@@ -131,6 +131,13 @@ type (
 		ParticipationUpdates
 	}
 
+	// TxnRoot represents the root of the merkle tree generated from the transactions in this block.
+	TxnRoot struct {
+		_struct          struct{}      `codec:",omitempty,omitemptyarray"`
+		DigestSha256     crypto.Digest `codec:"txn256"` // root of transaction vector commitment merkle tree using SHA256 hash function
+		DigestSha512_256 crypto.Digest `codec:"txn"`    // root of transaction merkle tree using SHA512_256 hash function
```

Comment:
These names don't seem helpful to me. DigestSha256 and DigestSha512_256 are implementation details.

Previously the name was simply `TxnRoot`. The name says nothing about the data except for what it represents. With the new names, there is nothing about what the data represents, just how it was implemented.

This is similar to the conversation we had about the REST APIs. I think this would be significantly clearer if the variables were named `MerkleTree` and `VectorCommitment`.

@algorandskiy I'm curious what you think about this too.

Reply:
I agree, `MerkleTree` or `MerkleTreeTxnRoot` sounds better

Reply:
I'm afraid it is more complicated than that. we need to remember that `TxnRoot` might also represent a flat commit (i.e hashing the concatenation of all the paysets in the block) . That was the implementation before on earlier protocol versions. Therefore I'm not sure using the word `merkleTree` or `VectorCommitment` is good. 

so we might want to change the structure name to `TxnCommitment` or similar, but those fields above are just the result of hash and cannot really state what was the input